### PR TITLE
implemented routing

### PR DESCRIPTION
**Pull request by dev-prabhat**
In this Branch I have implemented routing basic

Header.js
Navbar.js
Home.js
Label.js
Trash.js

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
small PR
Clean and understandable code
Good to merge

**Review by iRohitGaur — APPROVED** [review]
Clean code. Easy to understand. 

Good to merge 👍🏻

File: src/components/Header/header.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    grid-area: header;
+    padding: 0.5em 1.5em;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    box-shadow: 0px 0px 10px 2px var(--primary-theme-shade-20);
+}
+
```
Comment (bhushannaikaroskar) [inline src/components/Header/header.css:11]: In padding you can use px. Just a personal suggestion. in box-shadow you can use 0 instead of 0px

**Review by bhushannaikaroskar — APPROVED** [review]
Hey prabhat code is easy to read. Provided some minor suggestions. Rest I think everything is good. Approving the merge

**Pull request merged**